asn1ate/pyasn1gen.py

```diff
@@ -28,7 +28,7 @@
 import sys
 import argparse
 import keyword
-from asn1ate import parser, __version__
+from asn1ate import __version__
```

Comment:
Ah, `parser` gets accidentally imported `from asn1ate.sema import *` but explicit is indeed better than implicit so I get parser back. ;)